amqp.reflect/stack.yml

```diff
@@ -7,12 +7,12 @@ networks:
 services:
 
   zilla:
-    image: "ghcr.io/aklivity/zilla:develop-SNAPSHOT"
+    image: "ghcr.io/aklivity/zilla:latest"
```

Comment:
This should be reverted, some bindings are only present in `develop-SNAPSHOT` build due to incubator status, including `mqtt` and `amqp`.